src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -307,5 +307,12 @@ trait MarathonConf
     noshort = true,
     hidden = true,
     default = Some(MesosHeartbeatMonitor.DEFAULT_HEARTBEAT_FAILURE_THRESHOLD))
+
+  lazy val drainingTime = opt[Long](
+    "draining_time",
```

Comment:
Please add information to https://github.com/mesosphere/marathon/blob/master/docs/docs/command-line-flags.md#optional-flags-1